src/joints/rapier_generic_6dof_joint_3d.rs

```diff
@@ -126,11 +127,72 @@ impl RapierGeneric6DOFJoint3D {
         }
     }
 
+    fn update_joint(
```

Comment:
I left the existing implementation as is however we might want to store items in the rapier 6axis system in this struct already instead of having to convert in this method